server/channels/store/sqlstore/remote_cluster_store.go

```diff
@@ -191,6 +191,10 @@ func (s sqlRemoteClusterStore) GetAll(filter model.RemoteClusterQueryFilter) ([]
 		query = query.Where(sq.Eq{"rc.PluginID": filter.PluginID})
 	}
 
+	if filter.RequireOptions != 0 {
+		query = query.Where(sq.NotEq{fmt.Sprintf("(rc.Options & %d)", filter.RequireOptions): 0})
```

Comment:
hahaha, binary flags checking in the database, I don't know if it is cool or not, but I kinda like it.

Reply:
We'll be adding more flags for Miguel's project and this way we don't need to keep adding boolean columns.